doc/docs/1.13.x/deploy-manage/manage/s3gateway/configure-s3client.md

```diff
@@ -1,91 +1,102 @@
 
 1. Verify that the AWS CLI is installed:
 
-   ```shell
-   aws --version
-   ```
-
-1. Configure AWS CLI:
-
-   ```shell
-   aws configure
-   ```
-
-   **System Response:**
-
-   ```
-   AWS Access Key ID: YOUR-PACHYDERM-AUTH-TOKEN
-   AWS Secret Access Key: YOUR-PACHYDERM-AUTH-TOKEN
-   Default region name:
-   Default output format [None]:
-   ```
-
-   Both the access key and secret key should be set to your
-   Pachyderm authentication token. If authentication is not enabled
-   on the cluster, you can pass any value.
+      ```shell
+      $ aws --version
+      ```
+
+1. Configure AWS CLI. Use the `aws configure` command to configure your credentials file:
+      ```shell
+      $ aws configure --profile <name-your-profile>
+      ```
+      Both the access key and secret key 
+      should be set as mentioned in the [# Set Your Credentails](#set-your-credentials) section of this page.
+
+      **System Response:**
+      ```
+      AWS Access Key ID: YOUR-PACHYDERM-AUTH-TOKEN
+      AWS Secret Access Key: YOUR-PACHYDERM-AUTH-TOKEN
+      Default region name:
+      Default output format [None]:
+      ```
+!!! Note
+      Note that the `--profile` flag ([named profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)) is optional. If not used, your access information will simply be stored in the default profile. 
+      
+      To reference a given profile when using the S3 client, simply append `--profile <name-your-profile>` at the end of your command.
+
+!!! Info
+      Find **AWS S3 CLI** full documentation [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-services-s3-commands.html).
+ 
+## Configure boto3
+Before using Boto3, you need to [set up authentication credentials for your AWS account](#configure-the-aws-cli) using the AWS CLI as mentionned previously.
+
+Then follow the [Using boto](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#using-boto3) documentation starting with importing boto3 in your python file and creating your S3 ressource.
+   
+!!! Info   
+      Find **boto3** full documentation [here](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html).
+
+
+## Set Your Credentials
+- If [authentication is on](//TODO), 
+retrieve your session token in your active context:
+
+      ```shell
+      $ more ~/.pachyderm/config.json
+      ```
+      Search for your session token: `"session_token": "2a38f6685b7f40e2b56c67e4d93833b1"`.
+      **Make sure to fill both fields `Access Key ID` and `Secret Access Key` with that same value.**
+
+      Depending on your use case, it might make sense to pass the credentials of a robot-user or another type of user altogether. Refer to the [authentication section of the documentation](//todo) for more RBAC information.
+
+- If the authentication feature is not activated, make sure that whether you fill in those fields or not, their content always match. (i.e, both empty or both set to the same value)
```

Comment:
If authentication is disabled, make sure that all fields are set with the expected values, this could mean that some fields are empty.